SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialTypebindingManager.cpp

```diff
@@ -28,169 +30,200 @@ void USpatialTypebindingManager::Init(USpatialNetDriver* InNetDriver)
 		FMessageDialog::Debugf(FText::FromString(TEXT("SchemaDatabase not found! No classes will be supported for SpatialOS replication.")));
 		return;
 	}
-
-	FindSupportedClasses();
-	CreateTypebindings();
 }
 
-void USpatialTypebindingManager::FindSupportedClasses()
+FORCEINLINE UClass* ResolveClass(FString& ClassPath)
 {
-	TArray<FString> SupportedClassPaths;
-	SchemaDatabase->ClassPathToSchema.GetKeys(SupportedClassPaths);
-
-	for (auto& ClassPath : SupportedClassPaths)
+	FSoftClassPath SoftClassPath(ClassPath);
+	UClass* Class = SoftClassPath.ResolveClass();
+	if (Class == nullptr)
```

Comment:
Replace this with
```
checkf(Class, TEXT("Failed to load class at path %s"), *ClassPath);
```
The reason is that not resolving the class here is fatal, and we will crash because of it (and it's not worth guarding everywhere we call this)